src/cogs/abakus/abakus.py

```diff
@@ -87,6 +87,17 @@ async def post_dev_test(self, ctx):
 
         await post(self.channel, dev_event, self.client)
 
+    @commands.command()
+    @commands.has_role("Los Jefes")
+    async def post_updated(self, ctx):
+        """Post dev event.
+        
+        A dev event is an event starting in two hours, with registration opening in 11 minutes."""
+
+        dev_event: Event = Event(f"Dev eent{time.time()}", "Updated dummyevent", datetime.datetime.now(tz=local_timezone)+datetime.timedelta(hours=2), "Discord", datetime.datetime.now(tz=local_timezone)+datetime.timedelta(minutes=11), "https://abakus.no/events/2901","https://thumbor.abakus.no/40uu7jE2T02LcSrnDonxbJGSmd0=/0x500/Skjermbilde2021-05-1_sRKgopY.png")
```

Comment:
Typo

Reply:
Ment to be there. 
It tests that changed title doesn't cause new event to be posted, but rather changes already posted event

Reply:
Doesn't time.time() already account for this

Reply:
yes

Reply:
Call it supplementary